XenAdmin/MainWindow.Designer.cs

```diff
@@ -1308,6 +1309,7 @@ private void InitializeComponent()
             this.snapshotToolStripMenuItem,
             this.convertToTemplateToolStripMenuItem,
             this.exportToolStripMenuItem,
+            this.disableCbtToolStripMenuItem,

```

Comment:
Adding a new nemu item also requires the tests to be updated. See all the `VMToolStripMenuItem` usages in XenAdminTests\TabsAndMenus\MainMenuGeorge.cs